nix/lib/ci.nix

```diff
@@ -159,4 +159,13 @@ rec {
         )
         (builtins.attrNames set)
     );
+
+  # Takes an array of of systems and filters it by what the CI supports
```

Comment:
This isn't quite right: it outputs a subset of the "name -> string" attrset based on the argument, rather than filtering the argument, right?